relay-dynamic-config/src/feature.rs

```diff
@@ -22,6 +22,12 @@ pub enum Feature {
     /// Allow ingestion of metrics in the "custom" namespace.
     #[serde(rename = "organizations:custom-metrics")]
     CustomMetrics,
+    /// Enable extracting spans for modules in GA.
+    #[serde(rename = "projects:span-metrics-extraction-ga-modules")]
+    SpanMetricsExtractionGAModules,
```

Comment:
Keep in mind that we will have to keep these feature flags around, even if we make them the default at some point. I'd suggest some more functional naming for that, such as "db" instead of "GA".

Reply:
I had DB at first but we'll have more modules than this as time go by. I'm not sure DB would be a better name if we end up filtering on more than those modules.